examples/src/hello/mod.rs

```diff
@@ -16,7 +16,7 @@ async fn run() {
             .unwrap()
     };
 
-    log::info!("Selected adapter: {:?}", adapter.get_info())
+    println!("Selected adapter: {:?}", adapter.get_info())
```

Comment:
`println` won't output anything on the web/wasm targets, but log does

Reply:
Tentatively resolved with https://github.com/gfx-rs/wgpu/pull/5905#issuecomment-2206748367 (https://github.com/gfx-rs/wgpu/pull/5905/commits/5d57f0b5e2d4d944b86860b5cdbb25f73bf257ae).

Reply:
Ah, I figured there was some reason like this - and that if I filed a PR someone would tell me what it was:D